pkg/libmxnet.pkg

```diff
@@ -4,9 +4,9 @@ OPTS = dict(
     name = VAR.fullname,
     description = '''\
 Lightweight, portable deep learning library
- This package provides the core library exposing the MXNet C API.
- .
- It has been built with OpenCV and OpenMP support disabled.
+This package provides the core library exposing the MXNet C API.
+.
```

Comment:
See also: https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Description

However, this is the _control file_ spec, so I'm going to just quickly check if I get the same result if I don't manually include the `.` but place a blank line in the description.